subprojects/docs/src/samples/build-organization/gradle-plugin/tests/check.out

```diff
@@ -10,11 +10,3 @@
 > Task :greeting-plugin:compileTestJava
 > Task :greeting-plugin:processTestResources NO-SOURCE
 > Task :greeting-plugin:testClasses
-
-> Task :greeting-plugin:test
```

Comment:
❌ Can we add a `flags: "--warning-mode=none"` entry to the exemplar configuration instead?

Reply:
The output is not generated by the build run by the sample test runner, but from the build run by the GradleRunner inside the example.

I think the best fix would be to also remove the empty lines in the output normalizer. But I think that's defined somewhere in a different repository.

Reply:
IIRC all normalizers are in the gradle/gradle repo: https://github.com/gradle/gradle/blob/92877d51056dfac2c0a7512e43d769f12fbd1239/subprojects/docs/src/docsTest/java/org/gradle/docs/samples/ExemplarExternalSamplesFunctionalTest.java

Reply:
Thanks for the pointer. I had a look, but removing the newlines is not straight forward. The new line preceding a task name is added by Gradle as soon as the task prints out anything. So we can not strip it away in general, as it would also alter cases where a valid output is printed (like the samples with a _Hello World_ task). But we can also not match it easily as part of the deprecation warning as it is not directly connected to it.

E.g.:

```
> Task :greeting-plugin:testClasses

> Task :greeting-plugin:test
WARNING: An illegal reflective access operation has occurred
```

We can't change this generally. It would break expectations in other places. We would have to do some specific matching here if the task "only" prints the deprecation warning but nothing else and then change the whole output block belonging to that task.

I would tend to leave this alone here. In future Gradle versions, the warning will be gone and we won't have this particular problem anymore.

Reply:
Follow up: https://github.com/gradle/gradle/issues/14131